cmd/login/clientcert.go

```diff
@@ -11,8 +12,10 @@ import (
 	corev1alpha1 "github.com/giantswarm/apiextensions/v3/pkg/apis/core/v1alpha1"
 	"github.com/giantswarm/apiextensions/v3/pkg/label"
 	"github.com/giantswarm/backoff"
+	kubeconfig "github.com/giantswarm/kubeconfig/v3"
```

Comment:
I don't think we need this.
`"k8s.io/client-go/tools/clientcmd/api"` provides everything we need and we already use it in multiple places (even in this file)

Reply:
Cool, I'll use that one instead!